setup-multi-datacenter.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
```

Comment:
Move this into `examples/multi-dc/setup.sh` so we keep all example assets together.